library/include/modules/Units.h

```diff
@@ -68,6 +68,73 @@ static const int MAX_COLORS = 15;
  * The Units module - allows reading all non-vermin units and their properties
  */
 
+DFHACK_EXPORT bool isUnitInBox(df::unit* u,
+int16_t x1, int16_t y1, int16_t z1,
+int16_t x2, int16_t y2, int16_t z2);
```

Comment:
needs formatting (indenting? keep on one long line like some other functions in this file?

Reply:
weird, dunno how it lost its indentation